aip/0131.md

```diff
@@ -85,6 +85,36 @@ paths:
   - The variable for the resource ID **must** be named `id`. Variables
     representing the ID of parent resources **must** end with `Id`.
 
+#### RAML
+
+Single-resource `GET` methods **must** be specified in accordance with the
+following RAML spec.
+
+```yaml
+/publishers/{publisherId}/books/{bookId}:
+  uriParameters:
+    publisherId:
+      description: |
+        The unique identifier provided by the system for this publisher.
+    bookId:
+      description: |
+        The unique identifier provided by the system for this book.
+  get:
+    displayName: getBook
+    description: Get a single book.
+    responses:
+      200:
+        description: OK
+        body:
+          application/json:
+            type: Book
+```
+
+- The `displayName` **must** begin with the word `get` followed by the singular
+  form of the resource type's name.
+- The response content **must** be the resource itself.
+- The URI **must** contain a variable for each ID in the resource hierarchy.
```

Comment:
This one seems to conflict w the structure provided by the "name" field on most resources in proto-land. Do we have a plan for addressing that? (I'm not arguing for one or the other, just pointing out that they're different.)

Reply:
I'm not familiar with protobufs, so I'm not sure I am addressing this correctly:

I think the fact that this is in the RAML tab is how we might have seeming conflicting standards based on the protocol being used?